src/displayList.js

```diff
@@ -0,0 +1,41 @@
+import loadImage from './load-image-module';
+
+const displayList = (array) => {
+  console.log('hello');
+  const listContainer = document.getElementById('list-container');
+  listContainer.innerHTML = '';
+
+  for (let j = 0; j < 10; j += 3) {
+    const row = document.createElement('div');
+    row.classList.add('row');
+
+    for (let i = 0; i < 3; i += 1) {
+      const card = document.createElement('div');
+      card.classList.add('col-sm', 'card');
```

Comment:
I would recommend that:

- [ ] The page only **shows a single card** per row when the screen width is _less than 576px_ (`col` class in bootstrap).
- [ ] **show 2 cards** per row when the screen width is _less than 768px_ (`col-sm` class in bootstrap). 
- [ ] and **show 3 cards** per row when the screen width is _more than 768px_ (`col-md` class in bootstrap)

Reply:
`more than 768px` will show 4 cards instead of 3.